crest/Assets/Crest/Crest-Examples/BoatDev/Scripts/BoatControlPlayer.cs

```diff
@@ -0,0 +1,19 @@
+﻿using UnityEngine;
+using Crest;
+
+public class BoatControlPlayer : BoatControl
+{
+    [Tooltip("The input axis name for steering."), SerializeField]
+    string _steerInputAxisName = "Horizontal";
+
+    [Tooltip("The input axis name for throttle."), SerializeField]
+    string _throttleInputAxisName = "Vertical";
+
+    void Update()
+    {
+        var throttle = UnityEngine.Input.GetAxis(_throttleInputAxisName);
```

Comment:
The problem with this approach (it's a minor one, as this is demo-code and not meant to be a production-ready control systrem) is that [FixedUpdate runs before Update](https://docs.unity3d.com/Manual/ExecutionOrder.html) meaning that this system has an effective frame of latency.  I *don't* think this needs solving, but it could be worth adding a comment explaining this?

Reply:
Thanks. I've added a comment.